docs/proxy/google_serp_proxy/examples.md

```diff
@@ -0,0 +1,512 @@
+
+
+<marked-tab header="Node.js (got)" lang="javascript">
+const got = require("got");
+const HttpProxyAgent = require("http-proxy-agent");
+const HttpsProxyAgent = require("https-proxy-agent");
+
+// Replace <YOUR_PROXY_PASSWORD> below with your password
+// found at https://my.apify.com/proxy
+const proxyUrl = "http://groups-GOOGLE_SERP:<YOUR_PROXY_PASSWORD>@proxy.apify.com:8000"
+
+// Encode your query as a URI parameter
+const query = `q=${encodeURIComponent('Apple iPhone XS 64GB')}`;
+
+async function useProxy() {
+    const response = await got(`http://www.google.co.uk/search?${query}tbm=${'shop'}`, {
+        agent: {
+            http: new HttpProxyAgent(proxyUrl),
+            https: new HttpsProxyAgent(proxyUrl)
+        }
+    });
+    console.log(response.body)
+};
+
+useProxy();
+</marked-tab>
+
+
+<marked-tab header="Python 3" lang="python">
+import urllib.request as request
+import urllib.parse as parse
+
+# Replace <YOUR_PROXY_PASSWORD> below with your password
+# found at https://my.apify.com/proxy
+password = '<YOUR_PROXY_PASSWORD>'
+proxy_url = f"http://groups-GOOGLE_SERP:{password}@proxy.apify.com:8000"
+proxy_handler = request.ProxyHandler({
+    'http': proxy_url,
+})
+opener = request.build_opener(proxy_handler)
+
+query = parse.urlencode({ 'q': 'Apple iPhone XS 64GB', 'tbm': 'shop' })
+print(opener.open(f"http://www.google.co.uk/search?{query}").read())
+</marked-tab>
+
+
+<marked-tab header="Python 2" lang="python">
+import six
+from six.moves.urllib import request, urlencode
+
+# Replace <YOUR_PROXY_PASSWORD> below with your password
+# found at https://my.apify.com/proxy
+password = '<YOUR_PROXY_PASSWORD>'
+proxy_url = (
+    'http://groups-GOOGLE_SERP:%s@proxy.apify.com:8000' %
+    (password)
+)
+proxy_handler = request.ProxyHandler({
+    'http': proxy_url,
+})
+opener = request.build_opener(proxy_handler)
+query = parse.urlencode({ 'q': 'Apple iPhone XS 64GB', 'tbm': 'shop' })
+url = (
+    'http://www.google.co.uk/search?%s' %
+    (query)
+)
+print(opener.open(url).read())
+</marked-tab>
+
+
+<marked-tab header="PHP" lang="php">
+<?php
+$query = urlencode('Apple iPhone XS 64GB');
+$curl = curl_init('http://www.google.co.uk/search?tbm=shop&q=' . $query);
+curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+curl_setopt($curl, CURLOPT_PROXY, 'http://proxy.apify.com:8000');
+// Replace <YOUR_PROXY_PASSWORD> below with your password
+// found at https://my.apify.com/proxy
+curl_setopt($curl, CURLOPT_PROXYUSERPWD, 'groups-GOOGLE_SERP:<YOUR_PROXY_PASSWORD>');
+$response = curl_exec($curl);
+curl_close($curl);
+echo $response;
+?>
+</marked-tab>
+```
+
+## [](#using-the-apify-sdk) Using the Apify SDK
+
+If you're developing an actor using the [Apify SDK](https://sdk.apify.com), you can use Apify Proxy in:
+
+* [PuppeteerCrawler](https://sdk.apify.com/docs/api/puppeteer-crawler#docsNav) using the [createProxyConfiguration()](https://sdk.apify.com/docs/api/apify#apifycreateproxyconfigurationproxyconfigurationoptions) function.
```

Comment:
Wait, so does it work or not? Above it says that SERP does not work with Puppeteer.